package.json

```diff
@@ -62,12 +62,12 @@
     "less-loader": "^5.0.0",
     "style-loader": "^1.2.0",
     "stylelint": "^12.0.1",
-    "stylelint-config-wikimedia": "^0.8.0",
+    "stylelint-config-wikimedia": "^0.10.1",
     "stylelint-webpack-plugin": "^1.2.3",
     "terser-webpack-plugin": "^3.0.1",
     "webpack": "^4.43.0",
     "webpack-cli": "^3.3.11",
-    "webpack-dev-server": "^3.10.3"
+    "webpack-dev-server": "^3.11.0"
```

Comment:
What is the reason for upgrading this?

Reply:
I need to work on this more so that webpack-dev-server can be claimed as upgraded will make a separate PR for that later.